cdc/sink/mysql.go

```diff
@@ -765,28 +785,33 @@ func (w *mysqlSinkWorker) run(ctx context.Context) (err error) {
 		copy(rows, toExecRows)
 		err := w.execDMLs(ctx, rows, replicaID, w.bucket)
 		if err != nil {
-			w.txnWg.Add(-1 * txnNum)
 			txnNum = 0
```

Comment:
```suggestion
			txnNum = 0
```
could we remove the `txnNum`

Reply:
used in metrics